Bottom: 8faf883fe5de08676d01f7cd7448f3c5a0d673c5
Top:    69ac75bd63ef617622dd74a5e4e5c6fdfd15a0e4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 15:06:29 +0100

Refresh of doc-chmod-ugo-x-makefile-am

---

diff --git a/doc/Makefile.am b/doc/Makefile.am
old mode 100755
new mode 100644
diff --git a/doc/gs/Makefile.am b/doc/gs/Makefile.am
old mode 100755
new mode 100644
